CHANGES.rst

```diff
@@ -6,10 +6,41 @@
 Summary
 +++++++
 
-- The Gammapy v0.11 release is planned for March 2019.
+- Released on March 29th, 2019
+- 11 contributors
+- 2 months of work
+- 61 pull requests (not all listed below)
 
 **What's new?**
 
+Gammapy v0.11 implements a large part of the new joint-likelihood fitting
+framework, proposed in `PIG 7`_. This includes the introduction of the
```

Comment:
Do you mean PIG 8 here? Maybe use the word "dataset" in the first sentence to make it clear?
```
.. _PIG 7 - models: https://github.com/gammapy/gammapy/pull/1971
.. _PIG 8 - datasets: https://github.com/gammapy/gammapy/pull/1986
```

Reply:
Done